warehouse/models/rt_views/gtfs_rt_vs_sched_routes.sql

```diff
@@ -0,0 +1,69 @@
+WITH 
+-- selecting the distinct trips from GTFS Vehicle Postions for two operators (SamTrans 290 and Big Blue Bus 300) and two months
+vp_trips AS (
+    SELECT DISTINCT
+     calitp_itp_id,
+     calitp_url_number,
+     date AS service_date,
+     trip_id AS vp_trip_id,
+     trip_route_id,
+     vehicle_id
```

Comment:
If you don't use this column, I would drop it (BigQuery uses columnar storage so one of the easiest ways to improve query performance is to only select the set of columns that you specifically need.)